requirements.txt

```diff
@@ -5,3 +5,4 @@ uritemplate==0.6
 inflection==0.3.1
 requests==2.10.0
 mock==2.0.0
+PyJWT==1.6.4
```

Comment:
@rcbm unfortunately adding the dependency here is not going to do anything, since this is used to set up a django environment to run tests.

I don't like the way it's expected: https://github.com/ZeroCater/catering-core/blob/master/requirements.txt#L196 but that's how it needs to be done in order to work, I updated it in the branch we've been working on in `catering-core`

Reply:
@carllopez got it